recipes/fastk/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "FASTK" %}
+{% set version = "1.0" %}
+{% set sha256 = "80d483baa74afe491ed5a9b7444c5600d66c0b35fef82d48e5129aa90fb99e17" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/abueg/FASTK/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true [osx]
+  detect_binary_files_with_prefix: true
+
+requirements:
+  host:
+    - bzip2
+    - zlib
+    - libcurl==7.61.0
+    - make
+    - gxx_linux-64==5.4.0
```

Comment:
compiler should go into the `build` section

Reply:
moved! (also took out gxx_linux req. because that was me trying different things locally)